docs/locations-and-scheduling.md

```diff
@@ -29,77 +29,106 @@ The APIs used for Compute as a Service are:
   Locations are visible to users, but owned by the compute service team, i.e. read-only to the users and only projected
   into their workspaces for visibility. A placement decision references a location by name.
 
+  `SyncTarget`s in a `Location` are transparent to the user. Workload should be able to seamless move from one `SyncTarget` to another
+  within a `Location`. It is compute service's responsibility to ensure that:
+
+  1. Network connectivity: a workload in one `SyncTarget` can access service in another `SyncTarget` with a predefined DNS format.
```

Comment:
rather than "a DNS format" it is the Kubernetes service DNS format.

Maybe in general, mention that we strive for kube conformance for workloads in a location in the sense that for the user it looks like ONE cluster.